README.md

```diff
@@ -16,7 +16,7 @@ Using Reflections you can query for example:
   * Resources found in classpath  
 And more...
 
-*Reflections was written in the spirit of [Scannotations](http://bill.burkecentral.com/2008/01/14/scanning-java-annotations-at-runtime/) library*
```

Comment:
this should stay like it is. Because it's about the one single library named "Reflections" and not about multiple reflections.